app/renderer/components/tabs/tabPage.js

```diff
@@ -32,23 +32,6 @@ class TabPage extends React.Component {
     this.onMouseLeave = this.onMouseLeave.bind(this)
```

Comment:
ah, I think I accidentally grouped them together. Is there a test for it that is failing or did you just find it by manual testing?